src/index.js

```diff
@@ -3,8 +3,20 @@ import express from 'express';
 import mongoose from 'mongoose';
 
 import { DATABASE_URL, PORT } from './util/env';
-import { typeDefs } from './typeDefs';
-import { resolvers } from './resolvers';
+import { typeDefs } from './graphql/typeDefs';
```

Comment:
can you talk to me a little bit about the import for the env vars here? It is my experience that we would use 'dotenv' package and then require it at the top of the file. and then we would call the env vars using 'process.env.NAME_OF_VAR'

Reply:
It was a hack I did earlier to get it deployed. I'll implement dotEnv now and make another commit.